### PR TITLE
Add Stringtables

### PR DESCRIPTION
**Pull request by TyroneMF**
- Added stringtables to Ammo, Glocks and SCAR components.

### CODE REVIEW

File: addons/ammunition/CfgMagazines.hpp [hunk 1]
```diff
@@ -2,40 +2,40 @@ class CfgMagazines {
     class CA_Magazine;
 
     CLASS(8Rnd_P_000): CA_Magazine {
-        author = "TyroneMF";
+        author = $STR_tacgt_ammo_Author;
```
Comment (jonpas) [inline addons/ammunition/CfgMagazines.hpp]:
```suggestion
        author = $STR_TACGT_Ammo_Author;
```
Format for stringtables.

File: addons/ammunition/stringtable.xml [hunk 2]
```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project name="TACGT">
+    <Package name="Ammunition">
+        <Key ID="STR_tacgt_ammo_Author">
```
Comment (jonpas) [inline addons/ammunition/stringtable.xml]:
```suggestion
        <Key ID="STR_TACGT_Ammo_Author">
```

File: addons/ammunition/CfgMagazines.hpp [hunk 3]
```diff
@@ -2,40 +2,40 @@ class CfgMagazines {
     class CA_Magazine;
 
     CLASS(8Rnd_P_000): CA_Magazine {
-        author = "TyroneMF";
+        author = $STR_TACGT_Ammo_Author;
```
Comment (jonpas) [inline addons/ammunition/CfgMagazines.hpp]: This is pointless to stringtable, it's just names.

Comment (TyroneMF) [reply addons/ammunition/CfgMagazines.hpp]: Removed.

**Pull request merged**